.github/workflows/build.yml

```
@@ -44,6 +44,17 @@ jobs:
       - name: Prepare yarn 4
         run: corepack prepare yarn@4.9.1 --activate
 
+      - name: Compute app tree hash
+        id: app-tree-hash
+        run: echo "hash=$(git rev-parse HEAD:app)" >> $GITHUB_OUTPUT
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:1:42: Double quote to prevent globbing and word splitting

(shellcheck)